src/hooks/useVenuePoll.ts

```diff
@@ -0,0 +1,60 @@
+import { useMemo, useCallback } from "react";
+import firebase from "firebase/app";
+
+import { voteInVenuePoll, createVenuePoll, deleteVenuePoll } from "api/poll";
+
+import { PollMessage, PollValues, Vote } from "types/chat";
+
+import { useVenueId } from "./useVenueId";
+import { useUser } from "./useUser";
+
+export const useVenuePoll = () => {
+  const venueId = useVenueId();
+  const { user } = useUser();
+
+  const userId = user?.uid;
+
+  const voteInPoll = useCallback(
+    (votes: Vote[], pollId: string) => {
+      if (!venueId) return;
+
+      voteInVenuePoll({ venueId, votes, pollId });
```

Comment:
it should accept venueId, pollId, questionId, userId

Reply:
fixed in 0ec9686b7958dc8253722ffea7c1b5d62c52602e